src/snowflake/snowpark/mock/mock_plan.py

```diff
@@ -105,6 +110,35 @@ def output(self) -> List[Attribute]:
         return [Attribute(a.name, a.datatype, a.nullable) for a in self.attributes]
 
 
+class MockFileOperation(MockExecutionPlan):
```

Comment:
FileOperation shouldn't be part of an execution plan, otherwise we need to deal with duplicate operations.